compiler/src/dotty/tools/dotc/semanticdb/Tools.scala

```diff
@@ -8,6 +8,10 @@ import dotty.tools.dotc.semanticdb.Scala3.{_, given _}
 
 object Tools:
 
+  def mkURIstring(path: Path): String =
+    val uriParts = for part <- path.asScala yield new java.net.URI(null, null, part.toString, null)
```

Comment:
Anything wrong with path.toUri?

Reply:
`.toUri()` creates the absolute URI instead of a relative one which is used by semanticdb.

Reply:
Ok, sounds like something worth putting in a comment

Reply:
just updated the doc